### PR TITLE
Updated highest coverage check to only filter out future dates for specific variables

### PR DESCRIPTION
**Pull request by dwnoble**
* fixes bug where projected temperature readings were mistakenly being filtered  out

## before
uLcoMCqG4AUCiGi

## after
6WeUmhKL3darhKr (1)

### CODE REVIEW

File: server/lib/util.py [hunk 1]
```diff
@@ -253,6 +253,11 @@
     'chart_titles_by_sv.json', 'chart_titles_by_sv_sdg.json'
 ]
 
+# Filter out observations with dates in the future for these variable DCIDs
+# when finding the date of highest coverage
+FILTER_FUTURE_OBSERVATIONS_FROM_HIGHEST_COVERAGE_FOR_VARIABLE_DCIDS = set(
```
Comment (pradh) [inline server/lib/util.py:258]: frozenset

Comment (pradh) [inline server/lib/util.py:258]: can we do a slightly shorter name?

File: server/lib/util.py [hunk 2]
```diff
@@ -593,15 +598,21 @@ def _get_highest_coverage_date(observation_entity_counts_by_date,
     max_years_to_check: Only consider entity counts going back this number of
       years
   """
-  # Get observation dates in descending order, and filter out dates in the
-  # future (to exclude erroneous data)
+  # Get observation dates in descending order
   todays_date = str(date.today())
```
Comment (pradh) [inline server/lib/util.py:602]: move todays_date to under the if condition?

**Review by pradh — APPROVED** [review]
Thanks Dan!

File: server/lib/util.py (hunk 1 above)
Comment (dwnoble) [reply server/lib/util.py:258]: done

Comment (dwnoble) [reply server/lib/util.py:258]: changed to `FILTER_FUTURE_OBSERVATIONS_FROM_VARIABLES`

File: server/lib/util.py (hunk 2 above)
Comment (dwnoble) [reply server/lib/util.py]: done

**Pull request merged**